ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -348,22 +352,27 @@ private class SqlLedgerFactory(ledgerDao: LedgerDao) {
     @SuppressWarnings(Array("org.wartremover.warts.ExplicitImplicitTypes"))
     implicit val ec = DEC
 
-    def init(): Future[LedgerId] = startMode match {
+    def resetOrNot(): Future[Unit] = startMode match {
```

Comment:
Quite hard to name because of the branching, but `resetOrNot` sounds a bit vague about the cases in which it does not reset.

Reply:
What would you suggest. Rename to something like `resetIfNeeded`? Or maybe simply inline the code into the for-comprehension?